proto/storage/file_access.proto

```diff
@@ -52,4 +52,8 @@ message FileAccess {
   // The process that performed the action. May contain deployment/namespace
   // information.
   ProcessIndicator process = 5;
+
+  // Node information
+  string node_name = 6; // @gotags: search:"File Access Node,hidden"
+  string node_id = 7; // @gotags: search:"File Access Node ID,hidden"
```

Comment:
We explicitly do not want any search tags here (and I don't think these ones exist, either) it'll cause havoc with the augmentedobj construction later because we'll be combining a full `storage.Node` with `storage.FileAccess` so there'll be conflicts.

Also, could we have these as their own sub-message?

```
message Node {
    string id = 1;
    string name = 2;
}

// ...

Node node = 6;
```